views/MentionsAndSentiments/index.jsx

```diff
@@ -40,7 +39,12 @@ export default React.createClass({
             negative: item[2],
             neutral: item[3],
           })),
-      }));
+    }));
+
+    mentions = mentions.sort(function(a, b) {
```

Comment:
you could write this line as
`mentions = mentions.sort((a, b) => return (b.data.length - a.data.length))`